src/ray/protobuf/core_worker.proto

```diff
@@ -224,8 +223,7 @@ message RemoteCancelTaskRequest {
   bool recursive = 3;
 }
 
-message RemoteCancelTaskReply {
-}
+message RemoteCancelTaskReply {}
```

Comment:
Please avoid unnecessary diffs in the future.

Reply:
Ah my bad, looks like my Vim plugin for autoformatting protos is still turned on. Can do, although I think that we should have clang-format autoformat Protobuf files too as part of the format script.